components/Problem/index.tsx

```diff
@@ -33,7 +33,7 @@ Problem display can't be too long (>${MAXLEN}ch). \
             <Flex flex={1} justifyContent="flex-end">
               <Button 
                 size="sm" 
-                colorScheme={difficultyColors[problem.difficulty]} 
+                colorScheme={show.showDiff ? difficultyColors[problem.difficulty] : "purple"} 
```

Comment:
Maybe I would prefer to use a colour that isn't associated with a difficulty already (purple is associated with insane).

However, I don't mind this choice, so I'll approve this PR. It's up to you if you want to change this colour.

Reply:
Please change to grey.